landings/b/index.liquid

```diff
@@ -152,12 +153,28 @@ window.document.onload = () => {
     cta.forEach((c) => {
         c.addEventListener('click', () => {
             let searchParams = new URL(window.location.href).searchParams;
+			let w = window.open('_blank');
+            w.location = url;
             window.amplitude.track(`landing:b:click-apply`);
             let url = `https://waitlist.virto.network/?b=${searchParams.get('b')}`;
```

Comment:
if you want to override the url variable, you have to remove `let` and assign directly to the variable